data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+
+
+			label "tower"
+			`	The tower ahead of you is truly a sight to behold. It appears to be larger and taller than anything in human architecture, yet it looks very old - noticeably so - compared to the portions you've seen higher up in the clouds of dark smog. The base, which grows larger the higher up the tower goes, is lined with rows of torch-like structures with flames reaching high above them. The interior is just as intriguing - the first area you enter looks like it almost came straight out of some old medieval fantasy story. Everything looks rocky and sculpted in appearance, although some entrances seem to be made of newer metallic materials.`
+			`	You wonder if all Gegno architecture is built like this, but your curiosity is cut short when the Gegno, whom has been many steps ahead of you, grunts and gestures for you to enter a cage-like platform at the other side of an enormous room.`
+			choice
+				`	(Enter the cage.)`
+			`	Thankfully, the cage is not something like a jail cell, as with the clank of chains and other indeterminate sounds, you begin to ascend upwards to wherever your next destination may be. However, the shaking and clanging of the lift doesn't exactly ease your mind. All of a sudden, you start to hear a deep, methodical rumble coming from above you. In an instant you're met with a gust of air. Your cage is outside the building, rocketing upward with only the support of a few rails and pulleys. Surrounding the core of the tower in front of you are other large metal pulley systems, with the Gegno in one to your left in the distance. As you're ascending, you see that many Gegno are lined up on several balconies at different levels of the tower, playing large timpani-like drums to a tribal-sounding tune. Occasionally they yell or play a horn-like instrument.`
+			`	A few more minutes of drumming, several layers of smog, and possibly some height sickness all assail you, until the lift arrives at the very top of the tower - a huge, circular platform out in the open with a bunch of important-looking Gegno formed up around a large table, or display grid of sorts, with several models of planets and ships on top of it. A ring of electronic torches flanking each edge of the platform extends upward to the darkened sky.`
+			`	The Gegno that has guided you here gestures for you to go join them at their location amongst the others.`
+			choice
+				`	(Go to the Gegno.)`
+					goto "meeting"
+				`	(Stay where you are.)`
+
+			`	It's not long before a pair of Gegno soldiers walk over and essentially "pressure" you into joining the others. You're on their turf now, so you might as well play by their rules.`
```

Comment:
To me, this paragraph reads like a fourth wall break where Saugia personally addresses First Last and tells him to stop making bad choices. I think if you're going to keep it, it needs to be fleshed out a bit more, and more of a narration than a personal comment.